sdk/test-utils/perfstress/GettingStarted.md

```diff
@@ -0,0 +1,258 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+
+export abstract class `ServiceName`Test<TOptions> extends PerfStressTest<TOptions> {
+  serviceNameClient: ServiceNameClient;
+
+  constructor() {
+    super();
+    // Setting up the serviceNameClient
+  }
+
+  public async globalSetup() {
+    // .createResources() using serviceNameClient
+  }
+
+  public async globalCleanup() {
+    // .deleteResources() using serviceNameClient
+  }
+}
+```
+
+### [Test File](#Test-file)
+
+Following code shows how the individual perf test files would look like.
+
+```js
+import { ServiceNameClient } from "@azure/<service-sdk>";
+import { PerfStressOptionDictionary, drainStream } from "@azure/test-utils-perfstress";
+import { `ServiceName`Test } from "./serviceNameTest.spec";
+
+interface `ServiceNameAPIName`TestOptions {
+  newOption: number;
+}
+
+export class `ServiceNameAPIName`Test extends ServiceNameTest<`ServiceNameAPIName`TestOptions> {
+  // More details regarding the options in the next section
+  public options: PerfStressOptionDictionary<`ServiceNameAPIName`TestOptions> = {
+    newOption: {
+      required: true,
+      description: "A new option",
+      shortName: "sz",
+      longName: "newOption",
+      defaultValue: 10240
+    }
+  };
+
+  serviceNameClient: `ServiceName`Client;
+
+  constructor() {
+    super();
+    // Setting up the client
+  }
+
+  public async globalSetup() {
+    await super.globalSetup(); // Calling base class' setup
+    // Add any additional setup
+  }
+
+  async runAsync(): Promise<void> {
+    // call the method on `serviceNameClient` that you're interested in testing
+  }
+}
+```
+
+It is not mandatory to have separate base class and test classes. If there is nothing common among the testing scenarios of your service, feel free to merge base class with the test class to only have a single test class instead.
+
+### [Custom Options](#custom-options)
+
+As seen in the previous section, you can specify custom options along with the default options from the performance framework. You can access the options in the class using `this.parsedOptions`.
+
+Parsed options include the default options such as duration, iterations, parallel, etc offered by the perf framework as well as the custom options provided in the TestClass.
+
+```js
+interface `ServiceNameAPIName`TestOptions {
+  newOption: number;
+}
+
+export class `ServiceNameAPIName`Test extends ServiceNameTest<`ServiceNameAPIName`TestOptions> {
+  public options: PerfStressOptionDictionary<`ServiceNameAPIName`TestOptions> = {
+    newOption: {
+      required: true,
+      description: "A new option",
+      shortName: "sz",
+      longName: "newOption",
+      defaultValue: 10240
+    }
+  };
+
+  async runAsync(): Promise<void> {
+    // You can leverage the parsedOptions in the setup or globalSetup or runAsync methods as shown below.
+    // this.parsedOptions.duration.value!
+    // this.parsedOptions.newOption.value!
+  }
+}
```

Comment:
This is duplication for no good reason I think. Perhaps you can take this out from the previous section.